wm.c

```diff
@@ -123,16 +123,18 @@ static void ipc_cardinal_focus(long *d);
 static void ipc_cycle_focus(long *d);
 static void ipc_pointer_focus(long *d);
 static void ipc_config(long *d);
-static void ipc_save_monitor(long *d);
```

Comment:
I don't think I understand the rationale behind removing this functionality. `berry` has a system of associating workspaces to monitors. Multihead support is always going to be tricky, but I'm fairly happy with the functionality here. I'm certainly not opposed to adding new functionality or providing an alternative way of handling multiple monitors, but I do not think we should be removing existing functionality like this.